lexers/v/velocity.go

```diff
@@ -15,4 +25,28 @@ var Velocity = internal.Register(MustNewLexer(
 	Rules{
 		"root": {},
 	},
-))
+).SetAnalyser(func(text string) float32 {
+	var result float64
```

Comment:
Can't it be float32?

Reply:
Yeah, we had this issue before for another lexer. The reason to use `float64` here is that we do calculations below. The precision of `float32` is not enough here and will sometimes lead to wrong values (e.g. `1.50000001`).

Doing the float calculations with `float64` type and converting in the end to `float32`, solves this issue.